### PR TITLE
Value Reorganization

### PR DESCRIPTION
**Pull request by Geevies**
Deletes the worths files and moves them to their values instead, ctrl + click will now go to the correct definition in VSCode.

No user facing changes.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/weapons/material/material_weapons.dm [hunk 1]
```diff
@@ -23,6 +23,7 @@
 	var/default_material = DEFAULT_WALL_MATERIAL
 	var/material/material
 	var/drops_debris = TRUE
+	var/worth_multiplier = 1
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/material/material_weapons.dm:26]: DMDoc

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -39,6 +39,8 @@
 	var/germ_adjust = 0 // for makeshift bandages/disinfectant
 	var/carbonated = FALSE // if it's carbonated or not
 
+	var/value = 1
```
Comment (FluffyGhoster) [inline code/modules/reagents/Chemistry-Reagents.dm:42]: DMDoc

**Review by FluffyGhoster — APPROVED** [review]

**Comment by DreamySkrell** [comment]
based

**Pull request merged**